src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/formats/html/ModuleIndexWriter.java

```diff
@@ -84,7 +84,7 @@ protected void addIndex(Content target) {
 
         if (!groupModuleMap.keySet().isEmpty()) {
             TableHeader tableHeader = new TableHeader(contents.moduleLabel, contents.descriptionLabel);
-            Table table =  new Table(HtmlStyle.summaryTable)
+            var table =  new Table<ModuleElement>(HtmlStyle.summaryTable)
```

Comment:
let's remove one redundant space
```suggestion
            var table = new Table<ModuleElement>(HtmlStyle.summaryTable)
```